### PR TITLE
fix(footer): remove dead links and optimize

### PR DESCRIPTION
**Pull request by ckniffen**
## High Level Overview of Change

The documentation pages have changed over time:

- Carbon Calculator was absorbed by the impact page
- News redirects to homepage
- Wallets does not exist
- Exchange redirects to overview
- XRPL Businesses redirects to Uses

Also optimized the build by changing the source of the version number.

### Additional Context

Before the version number was populated by importing the entire package.json.  This caused caused the entire file to be included in the runtime javascript bundle.  Now it will populate a `VITE_*` environment variable that will be accessible at runtime saving 3kb (gzipped) in filesize.

### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)

## Before
Screenshot 2023-03-13 at 2 09 20 PM

## After

Monosnap XRPL Explorer | Ledgers 2023-03-21 17-37-52

### CODE REVIEW

**Comment by ckniffen** [comment]
Related to https://github.com/XRPLF/xrpl-dev-portal/issues/1720

File: src/containers/Footer/index.jsx [hunk 1]
```diff
@@ -24,31 +24,6 @@ const Footer = (props) => {
...
-          <a href="https://xrpl.org/wallet.html" className="footer-link">
-            Wallets
-          </a>
-          <a href="https://xrpl.org/exchanges.html" className="footer-link">
-            Exchanges
-          </a>
-          <a href="https://xrpl.org/businesses.html" className="footer-link">
-            XRPL Businesses
-          </a>
-          <a href="https://livenet.xrpl.org/" className="footer-link">
```
Comment (intelliot) [inline src/containers/Footer/index.jsx:49]: I think it's OK to leave this link here (or something similar) as it's just a convenient way to go back to the explorer homepage (when you've scrolled to the bottom of a page)

**Review by intelliot — APPROVED** [review]

File: src/containers/Footer/index.jsx (hunk 1 above)
Comment (ckniffen) [reply src/containers/Footer/index.jsx:49]: Should it go under build?

Comment (intelliot) [reply src/containers/Footer/index.jsx:49]: I think it's fine anywhere - whatever you like :)

**Review by JST5000 — APPROVED** [review]
LGTM

**Pull request merged**